latte/models/rlace/estimation.py

```diff
@@ -60,13 +65,25 @@ def fit(X: np.ndarray, Z: np.ndarray, rlace_params: Dict[str, Any], p_train: flo
     X_test_projected_A, X_test_projected_E = X_test @ A_hat_x, X_test @ E_hat_x
 
     # Construct the result
+    rng = np.random.default_rng(rng)
+    mi_estimation_ix = rng.choice(len(Z_test), min(8000, len(Z_test)), replace=False)
```

Comment:
This hard-coded `8000` looks a bit fishy. Can it be made a function parameter with a default value?

Reply:
Will do :)